schemas/json/wp-env.json

```diff
@@ -49,7 +49,7 @@
 					"default": []
 				},
 				"port": {
-					"description": "The primary port number to use for the installation. You'll access the instance through the port: 'http://localhost:8888'.",
+					"description": "The primary port number to use for the installation. You'll access the instance through the port: http://localhost:8888",
```

Comment:
This is a small improvement that should make the code editor suggest the correct URL to link to.

### Before

![image](https://github.com/user-attachments/assets/ebe81336-dbbf-4d8d-92fd-5e85cedb9623)

### After

![image](https://github.com/user-attachments/assets/024b1565-fc32-4416-8561-ca9da19007fb)